doc/feature/genericEventLoading.md

```diff
@@ -0,0 +1,119 @@
+Generic event loading {#genericEventLoading}
+=====================
+
+For research and debugging purposes, some times we need to quickly visualize
+a set of events in 3D space. For example, in a neural circuit with synaptic
+connectivity, we are interested in visualizing the position of the post-synaptic
+cells; or for validation purposes, we want to check that the LFP computation is
+done correctly for a predefined set of events.
+
+Since the variety of use cases is so wide, each of them with its own
+particularities, we need to find a generic way of loading any predefined set of
+events.
+
+The solution chosen for this feature is the loading of events from file. This
+offers users flexibility to generate all types of events in a simple and generic
+way, so the tool doesn't need to be adapted to fit the particular needs of any
+situation.
+
+
+## Requirements
+
+* Users should be able to specify their own input events, as easily as possible,
+from an external source (i.e. file).
+* The format of the input file should be clearly defined and documented.
+* All the library common capabilities should still be available for their use 
+with this feature. For example, generating 3D volumes with different
+resolutions and using any available functor.
+
+
+## API
+
+A TestLoader is already present in Fivox, which generates a set of 7 events in a
+fixed position, and is used for basic use cases such as the validation of the
+LFP computation in a test case. The input URI in this case is as follows:
+
+    "fivoxtest://?resolution=1&functor=lfp"
+
+What we need for the loading of any number of generic events is very similar to
+this, but we need a way to specify the source of our events, instead of
+generating them at runtime. We can use the BlueConfig path location as the event
+file path. A change in the name might also be convenient, as it is not anymore
+just for testing purposes. The loader and schema names are open for discussion.
+
+    "fivox:///absolute/path/to/events/file?resolution=1&functor=lfp"
+
+The path to the input file is optional, behaving like the original TestLoader
+and creating the initial 7 events if it is omitted.
```

Comment:
Can you point me out what are the differences and similarities between the proposed URL and the existing one using a BlueConfig?